app/views/exercises/implement.html.slim

```diff
@@ -6,16 +6,28 @@
 #editor-column
   - unless @embed_options[:hide_exercise_description]
     .exercise.clearfix
-      div
-        span.badge.rounded-pill.bg-primary.float-end.score
+      div.col-3.float-end
+        span.badge.rounded-pill.bg-primary.float-end.mt-2.score
+        - if current_contributor.programming_group?
+          button.btn.btn-sm.btn-primary.me-3.mt-1#start_chat  data= {url: "https://jitsi.fem.tu-ilmenau.de/openHPI_ProgrammingGroup#{current_contributor.id}"}
+            i.fa-solid.fa-video
 
-        h1 id="exercise-headline"
-          i id="description-symbol" class=(@embed_options[:collapse_exercise_description] ? 'fa-solid fa-chevron-right' : 'fa-solid fa-chevron-down')
-          = @exercise.title
+            span = t('exercises.editor.start_video')
+
+          div.small.text-body-tertiary.mt-1
+            == t('exercises.implement.external_data_policy', url:'https://www.tu-ilmenau.de/datenschutz')
+
+      h1 id="exercise-headline"
+        i id="description-symbol" class=(@embed_options[:collapse_exercise_description] ? 'fa-solid fa-chevron-right' : 'fa-solid fa-chevron-down')
+        = @exercise.title
 
       #description-card.lead class=(@embed_options[:collapse_exercise_description] ? 'description-card-collapsed' : 'description-card')
         = render_markdown(@exercise.description)
 
+      - if current_contributor.programming_group?
+        .small.text-body-tertiary
+          == t('exercises.implement.pair_programming_feedback', url:'https://etherpad.xopic.de/p/openHPI_PairProgrammingFeedback')
```

Comment:
```suggestion
          == t('exercises.implement.pair_programming_feedback', url: 'https://etherpad.xopic.de/p/openHPI_PairProgrammingFeedback')
```

Within the Etherpad, I would replace "Bugs" with something more generic (that is more easily understandable by our target group).